src/macros.rs

```diff
@@ -217,9 +217,9 @@ macro_rules! impl_pretty_debug {
     ($thing:ident) => {
         impl ::std::fmt::Debug for $thing {
             fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-                try!(write!(f, "{}(", stringify!($thing)));
+                r#try!(write!(f, "{}(", stringify!($thing)));
```

Comment:
don't like this `r#try!`, but never mind since `cargo fix` did that.
we can replace all `r#try!`  to `?` later.